CONTRIBUTING.md

```diff
@@ -36,6 +50,12 @@ formatters):
 pip install -r requirements-devel.txt
 ```
 
+> [!NOTE]
+> Any time you pull the latest fairseq2 commits from GitHub, make sure to re-run
+> the fairseq2n installation command above to get the most up-to-date binary. If
+> you observe runtime or test failures after the installation, it might be
+> because the latest nightlies are not published yet. If the problem persists
+> after about 12 hours, please create a GitHub issue.
```

Comment:
Nit but we can add a link to make the issue here

Reply:
Makes sense. Let me put a link tomorrow.

Reply:
Ok, made the changes along with PyTorch 2.1 updates. Please see the latest version of the doc.

Reply:
Thank you!